packages/fxa-auth-server/config/dev.json

```diff
@@ -2,7 +2,7 @@
   "contentServer": {
     "url": "http://localhost:3030"
   },
-  "customsUrl": "none",
```

Comment:
Just confirming that you want to enable rate limiting for local dev?

Reply:
Not necessarily. I needed it for testing here, but I can see how this could typically be annoying. I've reset the default to `none` and updated the docs to indicate that rate limiting is disabled by default for local dev.